knowledge_repo/app/templates/index-base.html

```diff
@@ -125,7 +125,9 @@
     } else {
       next_parameters["start"] = next_parameters["results"];
     }
-    $(".next-btn")[0].href += "?" + $.param(next_parameters);
+    if ($(".next-btn").length) {
+      $(".next-btn")[0].href += "?" + $.param(next_parameters);
```

Comment:
You need to make this explicitly `== 0`, take a look at the screenshot attached
<img width="562" alt="screen shot 2016-12-08 at 2 30 07 pm" src="https://cloud.githubusercontent.com/assets/4268311/21030334/e020256e-bd52-11e6-83b1-9881286d57c3.png">